<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Streaming Audio</string>
  </property>
  <property name="toolTipDuration">
   <number>8</number>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <item>
     <layout class="QVBoxLayout" name="playerLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">font: 57 11pt &quot;Roboto Medium&quot;;
border: 0;
</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="connectTab">
         <attribute name="title">
          <string>Connect</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="socketLabel">
            <property name="text">
             <string>Socket</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ipLabel">
            <property name="text">
             <string>IP Address</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="connectButton">
            <property name="styleSheet">
             <string notr="true">background-color: #524FA1;
color: white;</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_ip"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_port"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="playlistTab">
         <attribute name="title">
          <string>Playlist</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="playList">
            <property name="styleSheet">
             <string notr="true">border: 0;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: #524FA1;
color: white;</string>
            </property>
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="voiceTab">
         <attribute name="title">
          <string>Voice</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushToTalk">
            <property name="styleSheet">
             <string notr="true">background-color: #524FA1;
color: white;</string>
            </property>
            <property name="text">
             <string>Push to Talk</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>temp open udp</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settingsTab">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="volumeSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="artistLabel">
        <property name="styleSheet">
         <string notr="true">font: 57 8pt &quot;DINPro-Medium&quot;;
font: 57 14pt &quot;Roboto Medium&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Miley Cyrus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font: 57 8pt &quot;DINPro-Medium&quot;;
font: 25 italic 11pt &quot;Roboto Light&quot;;</string>
        </property>
        <property name="text">
         <string>Party in the U.S.A</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="audioCtrlLayout">
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playPauseButton">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="audioInfoLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="curCursorLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Roboto Light&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>0:28</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="maxCursorLabel">
          <property name="styleSheet">
           <string notr="true">font: 25 8pt &quot;Roboto Light&quot;;
color: rgb(161, 161, 161);</string>
          </property>
          <property name="text">
           <string>/ 3:25</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="cursorSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="userLayout">
      <item>
       <widget class="QLabel" name="userLabel">
        <property name="styleSheet">
         <string notr="true">font: 57 8pt &quot;DINPro-Medium&quot;;
font: 57 14pt &quot;Roboto Medium&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Users</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="userList">
        <property name="styleSheet">
         <string notr="true">font: 57 11pt &quot;Roboto Medium&quot;;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #524FA1;</string>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>&amp;Connections</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionJoin_Multicast"/>
   </widget>
   <widget class="QMenu" name="menuAudio_Test">
    <property name="title">
     <string>Audio Test</string>
    </property>
    <addaction name="actionPlaylist"/>
    <addaction name="actionCB"/>
    <addaction name="actionRecording"/>
   </widget>
   <addaction name="menuConnections"/>
   <addaction name="menuAudio_Test"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Request_Playlist">
   <property name="text">
    <string>&amp;Request Playlist</string>
   </property>
  </action>
  <action name="action_Visible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Visible</string>
   </property>
  </action>
  <action name="action_Host_Multicast">
   <property name="text">
    <string>&amp;Host Multicast</string>
   </property>
  </action>
  <action name="actionAqua">
   <property name="text">
    <string>Aqua</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionCrimson">
   <property name="text">
    <string>Crimson</string>
   </property>
  </action>
  <action name="actionChocolate">
   <property name="text">
    <string>Chocolate</string>
   </property>
  </action>
  <action name="actionDarkMagenta">
   <property name="text">
    <string>DarkMagenta </string>
   </property>
  </action>
  <action name="actionV">
   <property name="text">
    <string>DeepPink </string>
   </property>
  </action>
  <action name="actionForestGreen">
   <property name="text">
    <string>ForestGreen </string>
   </property>
  </action>
  <action name="actionAqua_2">
   <property name="text">
    <string>Aqua</string>
   </property>
  </action>
  <action name="actionRed_2">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlack_2">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionCrimson_2">
   <property name="text">
    <string>Crimson</string>
   </property>
  </action>
  <action name="actionChocolate_2">
   <property name="text">
    <string>Chocolate</string>
   </property>
  </action>
  <action name="actionDarkMegenta">
   <property name="text">
    <string>DarkMagenta</string>
   </property>
  </action>
  <action name="actionDeepPink">
   <property name="text">
    <string>DeepPink</string>
   </property>
  </action>
  <action name="actionForestGreen_2">
   <property name="text">
    <string>ForestGreen</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionWhite_2">
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="action_Modify">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="action_Advanced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="action_Folder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="action_Song">
   <property name="text">
    <string>Song</string>
   </property>
  </action>
  <action name="action_Default">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="action_Tiger">
   <property name="text">
    <string>Tiger</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionPlaylist">
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="actionJoin_Multicast">
   <property name="text">
    <string>Join Multicast</string>
   </property>
  </action>
  <action name="actionCB">
   <property name="text">
    <string>CB</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="text">
    <string>Recording</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
